pl_bolts/callbacks/data_monitor.py

```diff
@@ -97,7 +95,7 @@ def log_histogram(self, tensor: Tensor, name: str) -> None:
 
             logger.experiment.log(data={name: wandb.Histogram(tensor)}, commit=False)
 
-    def _is_logger_available(self, logger: LightningLoggerBase) -> bool:
+    def _is_logger_available(self, logger: Optional[LightningLoggerBase]) -> bool:
```

Comment:
Optional[LightningLoggerBase] is already set here.. couldn't reproduce this warning

Reply:
Oh, I didn't explain it clearly enough, sorry about that. The issue is that line 45 calls `self._is_logger_available(trainer.logger)`, where `trainer.logger` is `Optional[Logger]`, but your definition on this line is saying, that the first argument should be `Optional[LightningLoggerBase]`.

To be fair, `mypy` was complaining on line 45, but since you didn't change anything close to that, I couldn't comment on that specific line :slightly_smiling_face: